src/DucksNet.Infrastructure/Sqlite/EmployeeRepository.cs

```diff
@@ -27,7 +27,7 @@ public Result Delete(Employee entity)
 
     public Result<Employee> Get(Guid id)
     {
-        var employee = _databaseContext.Employees.Find(id);
+        var employee = _databaseContext.Employees.Find(id); //TODO (RO): nu aduce si relatia, inainte de punct; nu mapeaza pe officeId -> incarci relatia + obiectul respectiv
```

Comment:
This is more of a NOTE than a TODO. You could mark it as such and you;ll also fix sonarqube code smell